src/v2/Apps/Fair/Components/FairBooths.tsx

```diff
@@ -122,9 +124,8 @@ const FairBooths: React.FC<FairBoothsProps> = ({ fair, relay }) => {
       <Spacer mt={4} />
 
       <LoadingArea isLoading={isLoading}>
-        {fair.exhibitors?.edges!.map((edge, index) => {
-          const show = edge?.node!
-          if (!show.isDisplayable) {
+        {shows.map((show, index) => {
+          if (show.counts?.artworks === 0 || !show.partner) {
```

Comment:
[Previous implementation](https://github.com/artsy/force/pull/8192#discussion_r689447482) is used (`isDisplayable` is not suitable)